CITATION.cff

```diff
@@ -79,6 +79,11 @@ authors:
     email: omar.younis98@gmail.com
     affiliation: University of Bologna
     orcid: 'https://orcid.org/0009-0004-2783-7932'
+  - given-names: Roger
```

Comment:
For the paper here, we have only included authors that have provide significant API changes / support over the years so could you remove this please

Reply:
Done, and pushed